src/services/patientService.ts

```diff
@@ -36,6 +37,12 @@ const PatientService = {
   editPatient(patientId: string, patient: Patient) {
     return HttpService.patch<Patient>(baseEndPoint + '/patient/' + patientId, patient);
   },
+  getPatientsInCredit() {
```

Comment:
you don't need separate functions to 

1. get patients
2. get patients in credit
3. get patients in debt

You can put it in one function like this
```typescript
async getPatients(inCredit?: boolean, inDebt?: boolean) {
    const patients = await HttpService.get<Patient[]>(`${baseEndPoint}/patient${inCredit?'inCredit=true':''}${inDebt?'inDebt=true':''}`);
    return patients.map(patient => ({ ...patient, key: patient._id }));
  },
```

and by the way, because you used separate functions you forgot to map the patients and add them a key